docs/ci-cd/index.md

```diff
@@ -35,5 +35,12 @@ Work In Progress
 ## Tools
 
 -   [Daily Fail Summary](tools/daily_fail_summary.md)
+-   Spellcheck
+    -   Currently uses rojopolis/spellcheck-github-actions, a PySpelling-based
+        spellchecker.
+    -   This tool utilizes the definitions in .spellcheck.yml and
+        .github/.wordlist.txt to check all documentation files
+    -   Previously an additional spellchecker from reviewdog was used with
```

Comment:
I am unsure if we want this information here forever - it seems like we want to document what state things are and probably not document previous things (akin to not checking in commented-out code).

Reply:
I've removed references to unused tools.